kolibri/plugins/coach/assets/src/views/plan/assignments/RecipientSelector.vue

```diff
@@ -83,37 +82,64 @@
         default: new Array(),
       },
     },
+    data() {
+      return {
```

Comment:
This is the main piece of the refactor. I copy all the data from `AssignmentCopy/DetailsModal` into this component's `data` object and communicate changes to the Collection-level assignments through `watch.collectionIds`, but keep the same `updateLearners` event which watches for changes in `selectedLearnerIds` (which is two-way binded with IndividualLearnerSelected to make things cleaner).